Sources/SwerverCore/Server/ClientRequestHandler.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import Socket
```

Comment:
Is this a third party dependency? If so, it might be nice to wrap your use of `Socket` to isolate yourself from changes to the library.